pkg/services/rpcsrv/server.go

```diff
@@ -2635,6 +2650,7 @@ chloop:
 	// should be running concurrently to this one. And even if one is to run
 	// after unlock, it'll see closed s.shutdown and won't subscribe.
 	s.subsLock.Lock()
```

Comment:
Do you need `subsLock` here?

Reply:
Yes, see the https://github.com/nspcc-dev/neo-go/pull/2809#discussion_r1026114663.